index-fr.json

```diff
@@ -0,0 +1,737 @@
+"nique sa mère": 2,
+"nique ta mère": 2,
+"niquer": 2,
+"niquer sa mère": 2,
+"NTM": 2,
+"nul à chier": 2,
+"nulach": 2,
+"nulach’": 2,
+"oh hisse, enculé": 2,
+"on s’encule": 2,
+"ordure": 2,
+"oui ou merde": 2,
+"ouvrir sa grande gueule": 2,
+"pachole": 2,
+"panier": 2,
+"papier-cul": 2,
+"parachuter un Sénégalais": 2,
+"pare-choc": 2,
+"pare-chocs": 2,
+"parle à mon cul, ma tête est malade": 2,
+"partir en couille": 2,
+"pays-bas": 2,
+"PD": 2,
+"peau de zob": 2,
+"pédale": 2,
+"pédé": 2,
+"pédé comme un Grec": 2,
+"pédoque": 2,
+"peler": 2,
+"pelle à merde": 2,
+"pélot": 2,
+"penser avec sa bite": 2,
+"penser avec sa queue": 2,
+"pépom": 2,
+"pet sauce": 2,
+"péter à la gueule": 2,
+"péter dans la soie": 2,
+"péter le cul": 2,
+"péter les couilles": 2,
+"piment": 2,
+"pine": 2,
+"piner": 2,
+"pinocumettable": 2,
+"pinocumettre": 2,
+"pipe": 2,
+"pisse-au-lit": 2,
+"pisser": 2,
+"pisser à la raie": 2,
+"pisser de la copie": 2,
+"pisser sa côtelette": 2,
+"pisseur": 2,
+"plan cul": 2,
+"planter le javelot dans la moquette": 2,
+"plotte à cash": 2,
+"plume": 2,
+"poche": 2,
+"pogne-cul": 2,
+"poil": 2,
+"pomme à l’huile": 2,
+"pompe à merde": 2,
+"pomper": 2,
+"porc": 2,
+"porter le pet": 2,
+"pougner": 2,
+"pour les couilles du pape": 2,
+"pourrir": 2,
+"pousser le repas de la veille": 2,
+"pousser une gueulante": 2,
+"poutrer": 2,
+"PQR": 2,
+"putain": 2,
+"putain con": 2,
+"putain de": 2,
+"putain de bordel de merde": 2,
+"putain de merde": 2,
+"putain de moine": 2,
+"putain de sa mère": 2,
+"putain de sa race": 2,
+"putain de ta race": 2,
+"pute": 2,
+"pute borgne": 2,
+"pute de luxe": 2,
+"queue": 2,
+"queutard": 2,
+"qu’il aille se faire foutre": 2,
+"qu’ils aillent se faire foutre": 2,
+"qu’on l’encule": 2,
+"RAB": 2,
+"raccrocher à la gueule": 2,
+"radasse": 2,
+"ramoner": 2,
+"ramoner le trou de balle": 2,
+"ras-la-moule": 2,
+"ras la touffe": 2,
+"ras-le-bonbon": 2,
+"ras le cul": 2,
+"ras-les-fesses": 2,
+"ratisser le bunker": 2,
+"récurer la marmite": 2,
+"réempaffer": 2,
+"réenculer": 2,
+"refucker": 2,
+"remuer la merde": 2,
+"retard": 2,
+"retourne aux asperges": 2,
+"rien à enculer": 2,
+"rien à foutre": 2,
+"riflard": 2,
+"rondelle": 2,
+"roubignole": 2,
+"roupette": 2,
+"s’en beurrer la raie": 2,
+"sa mère": 2,
+"sac à merde": 2,
+"saint-chrême": 2,
+"saint-ciboire": 2,
+"saint-crême": 2,
+"saint-sacrament": 2,
+"saint-simonaque": 2,
+"salaud": 2,
+"salle des fêtes": 2,
+"salopard": 2,
+"salope": 2,
+"sans-couilles": 2,
+"sarce": 2,
+"s’archifoutre": 2,
+"s’astiquer le poireau": 2,
+"saute au paf": 2,
+"saute-au-paf": 2,
+"sauter": 2,
+"schleu": 2,
+"schnek": 2,
+"se battre les couilles de": 2,
+"se branler les couilles": 2,
+"se câlisser": 2,
+"se casser la gueule": 2,
+"se casser le cul": 2,
+"se casser le tronc": 2,
+"se casser les couilles": 2,
+"se cirer le manche": 2,
+"se cogner": 2,
+"se contrecrisser": 2,
+"se contrefoutre": 2,
+"se désenclaver la péninsule": 2,
+"se doigter": 2,
+"se faire enculer": 2,
```

Comment:
Here, right after line 623. I would add: 'se faire quelqu'un' or 'se faire' and give it a 2 or 1 since that would be an instance in which the verb do, would translate to a vulgar expression.

Reply:
change done